wire-ios-request-strategy/Sources/Synchronization/Decoding/EventDecoder+MLS.swift

```diff
@@ -67,8 +72,7 @@ extension EventDecoder {
         let decoder = EventPayloadDecoder()
         guard let payload = try? decoder
             .decode(Payload.UpdateConversationMLSMessageAdd.self, from: updateEvent.payload) else {
-            WireLogger.mls.error("failed to decrypt mls message: invalid update event payload")
-            return []
+            throw Failure.mlsDecodingError
```

Comment:
non-blocking: what about let it throw then like:

```suggestion
            let payload = try decoder
            .decode(Payload.UpdateConversationMLSMessageAdd.self, from: updateEvent.payload)
```

Reply:
Yeah I agree, especially because it likely contains helpful information.

Reply:
done in https://github.com/wireapp/wire-ios/pull/2416/commits/07c5bb080f856ea73eafcec4c09a8402d8bf55c4